alexkates/hashnode-next
# feat:add ld+json to /blog and /blog/[slug]

## What does this PR do?

Fixes #3 

## Type of change




## How should this be tested?
Repeat these steps for /blog and /blog/introducing-hashnode-next
- navigate to the url
- open dev tools and inspect page
- locate the script tag containing the ld+json
- confirm the details are correct

### Things to remember

- [x] Ran `pnpm format`
- [x] Ran `pnpm lint`
